Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -315,7 +317,15 @@ unsigned int compute2DCoords(RDKit::ROMol &mol,
     if (coordMap) {
       params.coordMap = *coordMap;
     }
-    return RDKit::CoordGen::addCoords(mol, &params);
+    unsigned int cid = RDKit::CoordGen::addCoords(mol, &params);
+    RDGeom::Point3D centroid = MolTransforms::computeCentroid(mol.getConformer(cid));
+    RDGeom::Transform3D trans;
+    trans.SetTranslation(-centroid);
+    RDGeom::POINT3D_VECT &locs = mol.getConformer(cid).getPositions();
+    for(auto atom: mol.atoms()) {
+      trans.TransformPoint(locs[atom->getIdx()]);
+    }
```

Comment:
Can be tons simpler:
```suggestion
    auto &conf = mol.getConformer(cid);
    MolTransforms::transformConformer(conf,trans);
```

Reply:
I thought there must be such  thing, but couldn't find it.